src/OrbitGl/CaptureWindowTest.cpp

```diff
@@ -32,74 +33,81 @@ constexpr int kViewportHeight = 100;
 constexpr double kSliderPosEpsilon = 0.0001;
 constexpr double kTimeEpsilonUs = 0.0000001;
 
-class NavigationTestCaptureWindow : public CaptureWindow, public testing::Test {
+class NavigationTestCaptureWindow : public testing::Test {
  public:
-  explicit NavigationTestCaptureWindow() : CaptureWindow(nullptr) {
-    capture_client_app_ = &capture_client_app_fake_;
-
-    Resize(kViewportWidth, kViewportHeight);
+  explicit NavigationTestCaptureWindow() : capture_window_{nullptr, &capture_client_app_fake_} {
+    capture_window_.Resize(kViewportWidth, kViewportHeight);
 
     capture_data_ = TrackTestData::GenerateTestCaptureData();
-    time_graph_ = std::make_unique<TimeGraph>(this, nullptr, &viewport_, capture_data_.get(),
-                                              &picking_manager_);
+    capture_window_.CreateTimeGraph(capture_data_.get());
 
     AddTimers();
-    PreRender();
-    time_graph_->ZoomAll();
+    capture_window_.PreRender();
+    capture_window_.GetTimeGraph()->ZoomAll();
```

Comment:
Starting with this change, all the changes in the file are rather mechanical. `PreRender` is replaced by `capture_window_.PreRender` and so on.